app/src/services/dataset.service.js

```diff
@@ -665,15 +665,15 @@ class DatasetService {
         };
         const createdDataset = await DatasetService.create(newDataset, user);
         logger.debug('[DatasetService]: Creating in graph');
-        if (stage !== 'staging') {
-            try {
-                await GraphService.createDataset(newDataset._id);
-            } catch (err) {
-                logger.error('Error creating widget in graph. Removing widget');
-                await createdDataset.remove();
-                throw new Error(err);
-            }
-        }
+        // if (stage !== 'staging') {
```

Comment:
`DatasetService.create(newDataset, user);` (line 666) already calls the graph, so this looks like duplicated stuff and is most likely not needed